src/ocaml/version.ml

```diff
@@ -47,3 +47,5 @@ let has_bigarray_library version = version < (5, 0, 0)
 let supports_alerts version = version >= (4, 8, 0)
 
 let has_sandboxed_otherlibs version = version >= (5, 0, 0)
+
+let has_META_files version = version >= (5, 0, 0)
```

Comment:
just adding a note in order not to forget - this is not part of 5.0, right?

Reply:
It is expected to land in 5.0.0~alpha1